docs/demand/faf.rst

```diff
@@ -21,12 +21,20 @@ datasets based on the relative tonnage of known modes (truck, rail, water) for e
 This proportional allocation ensures that demand for unknown modes is captured according to the
 transportation patterns of known modes.
 
+County-to-County Processing
+---------------------------
+The demand data is processed to generate county-to-county origin-destination pairs for each mode of transportation
```

Comment:
May be more accurate to say that FAF5 demand is projected from FAF regions into counties proportional to US census population.

Reply:
....assuming a uniform distribution of population across the county area (which is critical and could be improved). Arguably, we could even put a cute Sphinx annotation of some sort (`...note` https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html) that says: "There are other, more detailed, options for projecting FAF demand into counties and/or localizing the demand - but our data pipeline has opted for a simple, explainable process, especially for initial release."

Make this documentation useful! Fun! Use Sphinx! Enjoy!